integrations/slack

```diff
@@ -113,44 +101,112 @@ def generate_msg(alert):
     msg['ts'] = alert['id']
     attach = {'attachments': [msg]}
 
-    return json.dumps(attach)
+    json_msg = json.dumps(attach)
+    logger.debug("Message:\n%s", json_msg)
 
+    return json_msg
+
+
+def send_msg(msg: str, url: str):
+    logger.info("Sending message")
 
-def send_msg(msg, url):
     headers = {'content-type': 'application/json', 'Accept-Charset': 'UTF-8'}
     res = requests.post(url, data=msg, headers=headers)
-    debug(res)
+    if res.status_code == 200:
+        logger.info("Message sent successfully")
+    else:
+        raise requests.HTTPError("Failed sending message", res.reason)
+
+    logger.debug("Slack response:\nDate: %s\nStatus code: %d\nURL: %s\nSlack unique ID: %s",
```

Comment:
Perhaps we should encode the response and log it as is, adding multiline messages may make the log be harder to read. Let me know if you agree or if it's OK as it is.

Reply:
Encoding the JSON will not necessarily help in reducing the size of the log file compression will be needed after the encoding, and since that will require a decompression for reading it may cause complexity on the current issue.

It's a good subject to be aware of.

Reply:
I agree with converting the multiline log into a JSON. This can make it easier to create rules for users who want to monitor this log file.